arm-sql/2014-04-01/swagger/importExport.json

```diff
@@ -0,0 +1,495 @@
+                    "description": "Gets or sets the maximum size for the newly imported database."
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/ExportRequestParameters"
+                }
+            ],
+            "description": "Import Azure SQL Database parameters."
+        },
+        "ExportRequestParameters": {
+            "properties": {
+                "storageKeyType": {
+                    "type": "string",
+                    "description": "Gets or sets the type of the storage key to use. Valid values are StorageAccessKey and SharedAccessKey."
+                },
+                "storageKey": {
+                    "type": "string",
+                    "description": "Gets or sets the storage key to use."
+                },
+                "storageUri": {
+                    "type": "string",
+                    "description": "Gets or sets the storage uri to use."
+                },
+                "administratorLogin": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the SQL administrator."
+                },
+                "administratorLoginPassword": {
+                    "type": "string",
+                    "description": "Gets or sets the password of the SQL administrator."
+                },
+                "authenticationType": {
+                    "type": "string",
+                    "description": "Gets or sets the authentication type."
+                }
+            },
+            "description": "Export Azure SQL Database parameters."
+        }
+    },
+    "parameters": {
+        "SubscriptionIdParameter": {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "description": "Gets subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
+        },
+        "ApiVersionParameter": {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Client Api Version."
+        },
+        "ResourceGroupParameter": {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Resource Group to which the resource belongs.",
+            "x-ms-parameter-location": "method"
+        },
+        "ServerNameParameter": {
+            "name": "serverName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Azure SQL Server",
+            "x-ms-parameter-location": "method"
+        },
+        "ServerNameParameterGeneric": {
+            "name": "serverName",
```

Comment:
ServerNameParameterGeneric & ServerNameParamete both have the same value in the name property. One of them should be removed.